parent-pom.xml

```diff
@@ -28,6 +28,7 @@
     <bouncycastle.version>1.74</bouncycastle.version>
     <bouncycastle.bcfips.version>1.0.2.4</bouncycastle.bcfips.version>
     <bouncycastle.bcpkixfips.version>1.0.5</bouncycastle.bcpkixfips.version>
+    <bytebuddy.version>1.14.17</bytebuddy.version>
```

Comment:
you need to add it also for thin jar

Reply:
This should be actually in test scope, fixed it. Thin jar tests are going through fine for Java 21 & Mac.